audinterface/core/feature.py

```diff
@@ -96,38 +96,26 @@ class Feature:
             if features are extracted with a sliding window.
             If value is as a float or integer
```

Comment:
```suggestion
            If value is a float or integer
```

Please also change in all the other places.

Reply:
Thanks for spotting, I fixed it at all places.

Reply:
Thanks, can you please update the description.

Reply:
done